brian2tools/mdexport/mdexporter.py

```diff
@@ -90,9 +95,26 @@ def build(self, direct_call=True, debug=False, expander=None,
             print(self.md_text)
         elif self.filename:
             # start writing markdown text in file
-            md_file = open(self.filename + '.md', "w")
+            source_file = self.filename + ".md"
+            md_file = open(source_file, "w")
             md_file.write(self.md_text)
             md_file.close()
+            
+            # Check if Pandoc is installed
+            try:
+                subprocess.check_call(["pandoc"])
```

Comment:
This does not work, it blocks, since `pandoc` tries to read from `stdin`. Instead, I'd suggest going back to `pandoc --version`, but suppressing the output.

Reply:
```suggestion
                subprocess.check_call(["pandoc", "--version"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```